azext_edge/edge/providers/check/dataflow.py

```diff
@@ -53,6 +54,14 @@
 valid_source_endpoint_types = [DataflowEndpointType.kafka.value, DataflowEndpointType.mqtt.value]
 
 
+def _format_property_string(value: str, color: str = PROPERTY_DISPLAY_COLOR) -> str:
+    return COLOR_STR_FORMAT.format(value=value, color=color)
+
+
+def _basic_property_display(label: str, value: str, color: Optional[str] = PROPERTY_DISPLAY_COLOR, padding: int = PADDING) -> Padding:
```

Comment:
Yeah, these can probably be refactored to a common module - though I haven't looked to see where it can be utilized.

At the least I can move it to a better place in this PR and update the imports